modules/metering-store-data-in-s3.adoc

```diff
@@ -5,7 +5,12 @@
 [id="metering-store-data-in-s3_{context}"]
 = Storing data in Amazon S3
 
-Metering can use an existing Amazon S3 bucket or create a bucket for storage. Metering does not manage or delete any S3 bucket data. When uninstalling metering, any S3 buckets used to store metering data must be manually cleaned up.
+Metering can use an existing Amazon S3 bucket or create a bucket for storage.
```

Comment:
Did you mean _Metering can... create a bucket for storage._  Or _you can create a bucket for storage_?

Reply:
The odd part (to me), is that you have to manually specify where you want to use storage. As part of that, you have the option to use an existing bucket or tell metering to create a new bucket when you apply your config.